### PR TITLE
Feature/ruchita cm vis

### PR DESCRIPTION
**Pull request by RuchitaBK**
Sachin and i worked on few cm model part . please check this and review it .

### CODE REVIEW

File: ai4good/params/param_store.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def get_generated_disease_param_vectors(self) -> pd.DataFrame:
 class SimpleParamStore(ParamStore):
 
     def get_models(self) -> List[str]:
-        return ['compartmental-model']
+        return ['compartmental-model','Network-Model','Agent-Based Model']
```
Comment (vikrambajaj22) [inline ai4good/params/param_store.py:67]: can you make them lower case like the first one?

Comment (vikrambajaj22) [reply ai4good/params/param_store.py:67]: agent-based-model, network-model

Comment (RuchitaBK) [reply ai4good/params/param_store.py:67]: sure

File: fs/params/camp_params.csv [hunk 2]
```diff
@@ -19,4 +19,14 @@ Haman-al-Alil,30-39,0.42,1.85,30,9.49,,,,
...
+
+
+Mumbai,0-9,0.4,0.76,30,36.12,9465,,,
+Mumbai,10-19,0.25,0.81,30,27.81,,,,
+Mumbai,20-29,0.37,0.99,30,11.9,,,,
+Mumbai,30-39,0.42,1.85,30,9.49,,,,
+Mumbai,40-49,0.51,5.43,30,7.65,,,,
+Mumbai,50-59,0.59,15.05,30,4.32,,,,
+Mumbai,60-69,0.72,33.29,30,1.81,,,,
+Mumbai,70+,0.76,61.76,30,0.9,,,,
```
Comment (vikrambajaj22) [inline fs/params/camp_params.csv:32]: are these manually added? where do they come from?

File: ai4good/params/param_store.py (hunk 1 above)
Comment (RuchitaBK) [reply ai4good/params/param_store.py]: As per suggestions changes are done .

File: fs/params/camp_params.csv (hunk 2 above)
Comment (RuchitaBK) [reply fs/params/camp_params.csv:32]: yes i manually added this for testing purpose

File: ai4good/params/param_store.py (hunk 1 above)
Comment (vikrambajaj22) [reply ai4good/params/param_store.py]: thanks

File: fs/params/camp_params.csv (hunk 2 above)
Comment (vikrambajaj22) [reply fs/params/camp_params.csv:32]: ok, please remove them before merging

**Comment by RuchitaBK** [comment]
- Updated UI (Add Navbar in UI and Few UI changes for Run Queue Table.
-Remove Logarithmic scale from graph visualization.
-Updated graph

**Pull request merged**